lib/target/generic_cpp/generic_cxx_codegen.cc

```diff
@@ -566,6 +566,21 @@ void GenericCXXCodeGen::RunOnFunction(Function& function) {
         os_ << "odla_SetComputationItem(Comp, ODLA_OPT_BATCH_SIZE, "
                "(odla_item_value) &opt_batch_size);\n";
       }
+      if (opts_.bf16_mode == CodeGen::BF16Mode::Accuracy) {
```

Comment:
can we make a common helper function like "emitBF16Mode" for line line 569-583 and line 625-639?